vllm/config.py

```diff
@@ -2496,6 +2541,28 @@ def __post_init__(self):
             self.compilation_config.pass_config.enable_reshape = False
             self.compilation_config.level = CompilationLevel.PIECEWISE
 
+        if not envs.VLLM_USE_V1:
+            max_batchsize_to_capture = 0
+            if self.scheduler_config is not None and \
+                self.model_config is not None and \
+                    not self.model_config.enforce_eager:
+                max_batchsize_to_capture = \
+                    self.get_max_graph_batch_size(
+                    self.scheduler_config.max_num_seqs)
+            batch_size_capture_list = [
+                size for size in _BATCH_SIZES_TO_CAPTURE
+                if size <= max_batchsize_to_capture
+            ]
+        else:
+            batch_size_capture_list = []
+            if self.model_config is not None and \
+                not self.model_config.enforce_eager:
+                batch_size_capture_list = [1, 2, 4
+                                           ] + [i for i in range(8, 513, 8)]
+
+        self.compilation_config.init_with_cudagraph_sizes(
+            batch_size_capture_list)
```

Comment:
@WoosukKwon the logic is here.

Reply:
I see. Then, shouldn't we remove https://github.com/vllm-project/vllm/blob/4c05edb33ae4ae279421ddf981816d070e8ec37a/vllm/v1/worker/gpu_model_runner.py#L103?

Reply:
removed in [af2e5f0](https://github.com/vllm-project/vllm/pull/10838/commits/af2e5f04da2f8f6eeab513fa5b25197ab8cb884b)